docs/source/concept/how/BenchmarkProblems.rst

```diff
@@ -0,0 +1,41 @@
+.. _BenchmarkProblems:
+
+##################
+Benchmark problems
+##################
+
+To help choose between the different minimizers, FitBenchmarking
+comes with some curated Benchmark problems—although it’s straightforward
+add custom data sets to the benchmark, if that is more appropriate; see
+:ref:`problem_def` for specifics of how to add additional problems in a
+supported file format.
+
+We supply some standard nonlinear least-squares test problems in the
+form of the `NIST nonlinear regression set <https://www.itl.nist.gov/div898/strd/nls/nls_main.shtml>`_
```

Comment:
It would be nice to separate the sources of data similar to how the minimizers are